### PR TITLE
pass clip_norm to RMSprop

### PR DESCRIPTION
**Pull request by kylemcdonald**
I noticed that the clip_norm variable was unused, and I think this is how it was supposed to be used.

I've tested that this does not cause any obvious problems, but unfortunately can't verify that it helps because the original 900 epochs takes about 10 days on my machine.

### CODE REVIEW

**Comment by fchollet** [comment]
`clip_norm` is meant to prevent exploding gradients in RNNs. Here the RNN converge from the start, so `clip_norm` is not necessary (it wouldn't change anything).

**Comment by kylemcdonald** [comment]
interesting! i also didn't notice it changing anything, but figured it must be important since it's one of the hyper-parameters they optimized in the original paper. maybe it will be useful for another data set, or another configuration... but in that case, whoever is exploring will need to find the right value for themselves :)

**Comment by fchollet** [comment]
> figured it must be important since it's one of the hyper-parameters they optimized in the original paper

They did not use RMSprop in the original paper. SGD is notoriously unstable with RNNs and requires momentum, decay and gradient clipping to work. RMSprop or Adagrad work much more smoothly.

**Comment by kylemcdonald** [comment]
that makes a lot more sense, thanks for taking the time to explain!
